src/domain/schemas/user-schema.spec.js

```diff
@@ -9,14 +10,18 @@ import {
   postUserProfileBodySchema
 } from './user-schemas.js';
 
-const username = 'joedoe';
+const displayName = 'John Doe';
 const password = 'joedoe78';
 const email = 'joedoe78@gmail.com';
 
-const invalidUsernameCases = [
+const invalidDisplayNameCases = [
   {
-    description: 'username is shorter than min length',
-    body: { username: 'joedo' }
+    description: 'displayName is shorter than min length',
+    body: { displayName: [...Array(minDisplayNameLength - 1).keys()].join('') }
+  },
+  {
+    description: 'displayName is longer than max length',
+    body: { displayName: [...Array(maxDisplayNameLength + 1).keys()].join('') }
```

Comment:
As `maxDisplayNameLength` is > 9, the keys in this array start to be 2 characters long each. Probably better would be something like

~~~
Array.from({ length: maxDisplayNameLength + 1 }, () => 'x')

Reply:
That's a good point, I changed it as you mentioned and added the positive cases too (which would have caught the setup problem actually, if they had existed before)

Reply:
Cool, thanks!